templates/credential.html

```diff
@@ -7,6 +7,10 @@
 	<h2>{{ credential.name[LANG] }}</h2>
 	<p class="subtitle">Credential</p>
 
+	{% if credential.deprecated %}
+	<div class="alert alert-warning warning-test"><strong>This credential is deprecated</strong> since {{ credential.deprecated }}. Please do not use this credential.</div>
```

Comment:
It might be necessary for users to use the credential. For example, the pdbf email credential is deprecated. However, requestors are expected to still ask them for legacy instances wandering around in the wild still. Maybe we can say 'This credential is deprecated since (...). This means it cannot be obtained anymore. Please also provide an alternative when using this credential.' or something?

Reply:
Although we have to take into account that the deprecated time might be in the future of course...

Reply:
Good point, I have updated the text. As to deprecations in the future: what a requestor would have to do in that case is not really different from when the credential is already deprecated (in the past) - namely, offer an alternative in their session request, as you said.